### ISSUE TITLE
Documentation Issues 2024b

### ISSUE THREAD

**Issue by Sashadf1** [NREL/EnergyPlus#10605]
Issue overview (add additional documentation issues as separate comments below).
--------------
- [x] Typo in E+ documentation (I/O reference) for Site:GroundTemperature:BuildingSurface. Correct "The object is _options_ if you have no surfaces with ground contact." to instead say "The object is _optional_ if you have no surfaces with ground contact."

### Details
Some additional details for this issue (if relevant):
 - Version of EnergyPlus - 24.1

**Comment by mjwitte** [comment]
- [x] Documentation typo in 1.5.16 SimulationControl
Change "Or the user might have all flow rates and equipment sizes already specified and desire an annual weather without any preceding sizing calculations." to instead say "Or the user might have all flow rates and equipment sizes already specified and desire an annual weather simulation without any preceding sizing calculations."

TLDR there is a missing "simulation" noun.
(moved here from #10606)

**Comment by Sashadf1** [comment]
I will add any other documentation typos I find to this thread.

**Comment by mjwitte** [comment]
From helpdesk ticket 16741

> In the auxiliary Programs document, page 27 table 2.9, the shortname 'dirnozrad' is a typo. It's unclear if it should be 'dirnormrad' (which seemed to work when I used it) and is what is in figure 8. Figure 7 just has 'dirnorrad'. 

- [x] In table 2.9
dirnozrad --> dirnormrad
dirnorzillum --> dirnormillum

For reference, here are the header rows from an epw converted to the special csv format.
```
Date,HH:MM,Datasource,DryBulb {C},DewPoint {C},RelHum {%},Atmos Pressure {Pa},ExtHorzRad {Wh/m2},ExtDirRad {Wh/m2},HorzIRSky {Wh/m2},GloHorzRad {Wh/m2},DirNormRad {Wh/m2},DifHorzRad {Wh/m2},GloHorzIllum {lux},DirNormIllum {lux},DifHorzIllum {lux},ZenLum {Cd/m2},WindDir {deg},WindSpd {m/s},TotSkyCvr {.1},OpaqSkyCvr {.1},Visibility {km},Ceiling Hgt {m},PresWeathObs,PresWeathCodes,Precip Wtr {mm},Aerosol Opt Depth {.001},SnowDepth {cm},Days Last Snow,Albedo {.01},Rain {mm},Rain Quantity {hr}
Date,HH:MM,Datasource,Dry Bulb Temperature {C},Dew Point Temperature {C},Relative Humidity {%},Atmospheric Pressure {Pa},Extraterrestrial Horizontal Radiation {Wh/m2},Extraterrestrial Direct Normal Radiation {Wh/m2},Horizontal Infrared Radiation Intensity from Sky {Wh/m2},Global Horizontal Radiation {Wh/m2},Direct Normal Radiation {Wh/m2},Diffuse Horizontal Radiation {Wh/m2},Global Horizontal Illuminance {lux},Direct Normal Illuminance {lux},Diffuse Horizontal Illuminance {lux},Zenith Luminance {Cd/m2},Wind Direction {deg},Wind Speed {m/s},Total Sky Cover {.1},Opaque Sky Cover {.1},Visibility {km},Ceiling Height {m},Present Weather Observation,Present Weather Codes,Precipitable Water {mm},Aerosol Optical Depth {.001},Snow Depth {cm},Days Since Last Snow,Albedo {.01},Liquid Precipitation Depth {mm},Liquid Precipitation Quantity {hr}
```

- [x] Also, throughout the auxiliary programs doc, every occurrence of `\_` should be simply `_` when it appears in a code list format. Searching the pdf shows these. I'm not sure if that sequence appears correctly elsewhere in the source file(s). e.g. from p. 34, Figure 6. DEF file for formatted custom file.

![image](https://github.com/user-attachments/assets/e75bb655-737a-436b-9a0c-d134167dd2aa)

**Comment by mjwitte** [comment]
- [x] From Helpdesk ticket 16744.
 The Input Output Reference for ChillerHeaterPerformance:Electric:EIR says:

> 1.23.30.18 Field: Condenser Type
This alpha field contains the type of condenser. WaterCooled is the only valid type, and it requires the
full specification of the condenser loop and its associated equipment.

But the IDD shows two options:
```
  A3 , \field Condenser Type
       \type choice
       \key AirCooled
       \key WaterCooled
       \default WaterCooled
 ```

**Comment by jmarrec** [comment]
* [x] ScheduleFile Column Separator mentions Fixed, but IDD says Space

https://github.com/NREL/EnergyPlus/blob/a7fe8f76173de1c019072ac1bfe24b0d6bf4a037/doc/input-output-reference/src/overview/group-schedules.tex#L489-L491

```
Schedule:File,
[...]
  A4 , \field Column Separator
       \type choice
       \key Comma
       \key Tab
       \key Space
       \key Semicolon
       \default Comma
```

**Comment by rraustad** [comment]
- [x] IO Ref Sizing:Zone idf snippet is out of date. Does not show SAT Input Method or Temperature Difference inputs:

    Sizing:Zone ,
    SPACE5 -1, !- Name of a zone
    14., !- Zone cooling design supply air temperature {C}
    50., !- Zone heating design supply air temperature {C}
    0.009, !- Zone cooling design supply air humidity ratio {kg-H2O/kg-air}
    0.004, !- Zone heating design supply air humidity ratio {kg-H2O/kg-air}
    DSOA1 , !- Design Specification Outdoor Air Object Name

**Comment by mjwitte** [comment]
- [x] In the EMS Application Guide, the examples show an old form for `EnergyManagementSystem:OutputVariable` where the units are part of the variable name.

![image](https://github.com/user-attachments/assets/1004e966-ccb7-4cb1-b3f9-7f8e84aeece6)

Using this form results in a warning:
![image](https://github.com/user-attachments/assets/1ae90af3-ccc4-4598-a6ef-1779d759969c)

The current object has a separate field for units.
```
  EnergyManagementSystem:OutputVariable,
    Weighted Average Building Zone Air Temperature,  !- Name
    AverageBuildingTemp,     !- EMS Variable Name
    Averaged,                !- Type of Data in Variable
    ZoneTimeStep,            !- Update Frequency
    ,                        !- EMS Program or Subroutine Name
    C;                       !- Units
```

Ref helpdesk ticket 16759.

**Comment by mjwitte** [comment]
- [x] See #10833

**Comment by mjwitte** [comment]
- [x] See #10835

**Comment by Sashadf1** [comment]
- [x] ![image](https://github.com/user-attachments/assets/47549317-984e-4499-a947-59d2543793e3)

Should say "0.0, the local wind speed..." for full sentence.

**Comment by RKStrand** [comment]
- [x] See also https://github.com/NREL/EnergyPlus/issues/10674

**Comment by RKStrand** [comment]
- [x] Also look at https://github.com/NREL/EnergyPlus/issues/10364
